translations/en.json

```diff
@@ -2318,5 +2318,17 @@
     "emoji-symbols": "Symbols",
     "emoji-flags": "Flags",
     "emoji-no-results-title": "No emojis match your search",
-    "emoji-no-results-description": "Try something like “rainbow”"
+    "emoji-no-results-description": "Try something like “rainbow”",
+    "other-options": "Other options",
+    "enter-keycard-pin": "Enter Keycard PIN",
+    "alisher-card": "Alisher Card",
```

Comment:
Alisher is one of the designers :) this should be something like named card
`"card-with-name": "{{name}} Card",`

Reply:
@J-Son89 Updated!